doc/source/examples/incompressible-flow/2d-naca0012-low-reynolds/2d-naca0012-low-reynolds.rst

```diff
@@ -0,0 +1,291 @@
+
+Parameter file
+----------------
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~
+
+The problem is transient. An adaptative time step is used to ensure that the CFL:math:`<10` condition is obtained. This boundary for the CFL number was chosen to get relevant results while having reasonable simulation times. The reader is refered to the :doc:`../2d-lid‐driven-cavity-flow/lid‐driven-cavity-flow` example for a better understanding of the time step adaptation to kep the CFL below a certain treshold. A second order backward differentiation (BDF2) scheme was chosen.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method            = bdf2
+      set output name       = naca-output
+      set output path       = ./output/
+      set output frequency  = 50
+      set adapt             = true
+      set max cfl           = 10
+      set time end          = 40
+      set time step         = 0.01
+    end
+    
+Physical properties 
+~~~~~~~~~~~~~~~~~~~~~
+
+In this problem, the Reynolds number is defined as:
+
+.. math::
+        Re = \frac{u_{\infty}C}{\nu}
+
+where :math:`C` is the chord length of the airfoil, :math:`u_{\infty}` is the magnitude of the inlet velocity (free-stream velocity) and :math:`\nu` the kinematic viscosity. :math:`C` and :math:`u_{\infty}` are taken equal to 1. Then, in order to get :math:`Re = 1000` one must have :math:`\nu = 0.001`.
+
+.. code-block:: text
+	
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 0.001
+      end
+    end
+
+
+Forces
+~~~~~~~~~~~~~~~~~~
+
+The forces are computed in order to obtain the drag and lift coefficients later on:
+
+.. code-block:: text
+	
+    subsection forces
+      set verbosity             = verbose
+      set calculate force       = true
+      set calculate torque      = false
+      set force name            = force
+      set output precision      = 10
+      set calculation frequency = 1
+      set output frequency      = 1
+    end
+    
+Mesh 
+~~~~~~~~~~~~~~~~~~
+
+A C-Type mesh was created around the NACA. It is one of the usual types of mesh chosen for airfoils because it allows for the curvature of the grid to match the leading edge of the airfoil. Also, it allows the user to place more cells in areas which require higher resolution, typically near the upper and lower surfaces of the airfoil and in the wake. These properties are also true for O-type meshes, the difference being that the exterior boundary is a full circle, instead of a semi-circle in entry. In order to obtain such a mesh, the Transfinite functions of ``gmsh`` were used. To get a good understanding of how it was done, the reader is advised to read through the ``cTypeMesh.geo`` file included in the example which is thoroughly commented. To generate a mesh with a different angle of attack, the only thing required is to change the ``angle`` parameter in the ``cTypeMesh.geo`` file.
```

Comment:
```suggestion
A C-Type mesh was created around the NACA. It is one of the usual types of mesh chosen for airfoils because it allows for the curvature of the grid to match the leading edge of the airfoil. Also, it allows the user to place more cells in areas which require higher resolution, typically near the upper and lower surfaces of the airfoil and in the wake. These properties are also true for O-type meshes, the difference being that the exterior boundary is a full circle, instead of a semi-circle in entry. In order to obtain such a mesh, the Transfinite functions of ``gmsh`` were used. To get a good understanding of how it was done, the reader is advised to read through the ``cTypeMesh.geo`` file included in the example, which is thoroughly commented on. To generate a mesh with a different angle of attack, the only thing required is to change the ``angle`` parameter in the ``cTypeMesh.geo`` file.
```